src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -111,6 +112,7 @@ public HttpRequestHeaders Headers
 
         internal bool HasHeaders => _headers != null;
 
+        [Obsolete("Use Options instead.")]
         public IDictionary<string, object?> Properties
```

Comment:
I believe the intention is this implementation should be changed to be:
```C#
public IDictionary<string, object?> Properties => Options;
```

Reply:
Addressed in recent commits